e2e/old-arch/panels-suite/geomap-spatial-operations-transform.spec.ts

```diff
@@ -2,7 +2,8 @@ import { e2e } from '../utils';
 
 const DASHBOARD_ID = 'P2jR04WVk';
 
-describe('Geomap spatial operations', () => {
+// Flaky test - skipping for now
+describe.skip('Geomap spatial operations', () => {
```

Comment:
this one also fails 😢 

https://github.com/grafana/grafana/blob/main/e2e/panels-suite/geomap-spatial-operations-transform.spec.ts#L5

Reply:
See: https://github.com/grafana/grafana/pull/104155